src/bundle/Resources/config/services/permissions.yml

```diff
@@ -0,0 +1,11 @@
+services:
+    _defaults:
+        autowire: true
+        autoconfigure: true
+        public: false
+
+    EzSystems\EzPlatformAdminUi\Permission\:
+        resource: "../../../lib/Permission"
```

Comment:
Quite risky. Given `Permission` is a broad namespace you might not want every class to become a service.